sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -1436,15 +1436,17 @@ class DataFrame private[sql](
   ////////////////////////////////////////////////////////////////////////////
 
   /**
-   * Save this [[DataFrame]] to a JDBC database at `url` under the table name `table`.
+   * Save this [[DataFrame]] to a JDBC database at `url` under the table name `table`
+   * and connection propeties optionally passed in `properties`.
    * This will run a `CREATE TABLE` and a bunch of `INSERT INTO` statements.
    * If you pass `true` for `allowExisting`, it will drop any table with the
    * given name; if you pass `false`, it will throw if the table already
    * exists.
    * @group output
    */
-  def createJDBCTable(url: String, table: String, allowExisting: Boolean): Unit = {
-    val conn = DriverManager.getConnection(url)
+  def createJDBCTable(url: String, table: String, allowExisting: Boolean,
+      properties: Properties = new Properties()): Unit = {
```

Comment:
Also, default arguments do not work for java so you should create a separate method.